pkg/azdiskschedulerextender/azdiskschedulerextender_metrics.go

```diff
@@ -51,6 +51,20 @@ var (
 		},
 		[]string{"filter"})
 
+	SchedulerExtenderRequestCount = metrics.NewCounter(
+		&metrics.CounterOpts{
+			Namespace: scheduler,
+			Name:      "extender_requests_total",
+			Help:      "Total extender request.",
+		})
+
+	// PrioritizeNodesRequestCount = metrics.NewCounter(
```

Comment:
Will be removing `PrioritizeNodesRequestCount` and `CacheSize` as those are not used for the scheduler extender.